manifests/0000_26_cloud-controller-manager-operator_19_networkpolicy.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: ccmo-network-policy
```

Avoid abbreviation in resource names like this. You also don't need the `network-policy` bit as that's obvious from the resource type
```suggestion
  name: cloud-controller-manager-operator
```